packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -629,9 +633,40 @@ class NavigationColumn extends ColumnData<LinkData>
     bool isRowHovered = false,
     VoidCallback? onPressed,
   }) {
-    return isRowHovered
-        ? const Icon(Icons.arrow_forward)
-        : const SizedBox.shrink();
+    return IconButton(icon: Icon(Icons.arrow_right), onPressed: () {
+      showDialog(
+        context: context,
+        builder: (_) {
+          return DevToolsDialog(
+            title: Text('Please enter parameters'),
+            content: Column(
+              children: [
+                Row(
+                  children:[
+                    Column(
+                      children: [
+                        Row(children: [_wrap(Text('id:')), _wrap(Text('34'))]),
+                        Row(children: [_wrap(Text('bookId')), _wrap(Text(''))]),
+                      ],
+                    ),
+                    Column(
+                      children: [
+                        Text('Query parameters'),
+                        Row(children: [_wrap(Text('view')), _wrap(Text('showDetails'))]),
+                      ],
+                    )
+                  ],
+                ),
+                Text('Result url: /user/34/books/:bookId?view=showDetails')
```

Comment:
is this stubbed data that needs to be removed?

Reply:
reverted